sync-core/src/test/java/com/cloudant/common/ProxyTestBase.java

```diff
@@ -0,0 +1,47 @@
+package com.cloudant.common;
+
+import com.cloudant.http.Http;
+import com.cloudant.mazha.CouchConfig;
+import com.cloudant.mazha.SpecifiedCouch;
+import com.cloudant.sync.replication.ReplicationTestBase;
+import com.cloudant.sync.util.Misc;
+
+import java.net.URL;
+
+/**
+ * Created by tomblench on 23/07/15.
+ */
+public class ProxyTestBase extends ReplicationTestBase {
+
+    // test.couch.port becomes proxyPort 'this is where toxiproxy listens'
+    // test.couch.proxy.admin.port
+    // test.couch.proxy.target.port
+
+    // TODO set these up through config
```

Comment:
I think you should add `System.getProperty("test.couch.port",8000)` type calls here as the properties already appear to be in the gradle. Although I also think the properties should be customizable in the gradle following the usual pattern.